forage-android/src/main/java/com/joinforage/forage/android/core/ForageSDKInterface.kt

```diff
@@ -98,7 +98,7 @@ internal interface ForageSDKInterface {
  * to create multiple payments, set to true by default.
  */
 data class TokenizeEBTCardParams(
-    val foragePanEditText: ForagePANEditText,
+    val foragePanEditText: ForagePanElement,
```

Comment:
Is this rename from `ForagePANEditText` -> `ForagePanElement` a breaking change for ecom?

Reply:
Nope! `ForagePANEditText` is not getting renamed at all! Instead, `ForagePanElement` is a new abstract class that is being inserted in the class hierarchy right between `AbstractForageElement` and `ForagPANEditText`.

*Previous Hierarchy*
* `AbstractForageElement`
  * `ForagePANEditText`
  * `ForagePINEditText`

*New Hierarchy*
* `AbstractForageElement`
  * `ForagePanElement` 
    * `ForagePANEditText`
  * `ForagePinElement` 
    * `ForagePINEditText`

Inserting `Forage*Element` between `AbstractForageElement` and `Forage*EditText` will allow common requirements of PIN and PAN to live in `Forage*Element` while diverge requirements for ecom and pos can define these in `Forage*EditText`, which inherit from `Forage*Element`

So change params we expose in the public contract to point from specific `Forage*EditText` -> abstract `Forage*Element`, won't be a breaking change, fortunately!

Reply:
Thank you for the clarification.